### DISCUSSION TITLE
Plug in Bug

### DISCUSSION THREAD

**Discussion by sukchanun** [community/community#116874] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If i setting volume over 200% plugin gonna bug
![Screenshot 2024-04-05 143852](https://github.com/community/community/assets/86426117/cd2a0ee3-5dbb-423e-aefc-9b2ca371b631)

**Comment by sukchanun** [comment]
FR

**Comment by Akash1134** [comment]
Hi @sukchanun ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

If you have not clearly explained your issue and it is related to GitHub, please post a new dicussion. 

Best of luck!
